### ISSUE TITLE
About learning rate

### ISSUE THREAD

**Issue by nnks-dev** [ibalazevic/multirelational-poincare#1]
How was the learning rate determined?

**Comment by ibalazevic** [comment]
Please refer to the paper for all the info on hyperparameter tuning.

**Comment by nnks-dev** [comment]
Thank you very much. Let me ask you another question.
 I refer to Appendix E for details of the projection method, but I do not know the specific execution method. What kind of code should be executed on Python?

**Comment by nnks-dev** [comment]
And how can I get a diagram?

**Comment by ibalazevic** [comment]
This code is not a part of the main algorithm, so I haven't included it in the github repo. The projection method should be relatively straightforward to implement from the formulas in the paper once you have the trained embeddings and for plotting we used standard matplotlib methods.
